### ISSUE TITLE
not operator matches too broadly

### ISSUE THREAD

**Issue by robbi5** [sass/libsass#873]
Hi,
`not` in declarations gets converted into `false`, even when other characters are appended.

Example:
```sass
div {
  foo: note; }

div {
  foo: note 1; }
``` 

Expected output:
```sass
div {
  foo: note; }

div {
  foo: note 1; }
```

Actual output:
```sass
div {
  foo: note; }

div {
  foo: false 1; }
```

(Our use case: We have a list of icons, with icons named like "icon-note", "icon-guitar", ...)

**Comment by xzyfer** [comment]
@robbi5 this case is fix on master and will be 3.2.

@rawalter21 this appears to still be an error. Can you please create a sass-spec? Here's an example of a good sass-spec PR https://github.com/sass/sass-spec/pull/239

**Comment by satya164** [comment]
Similar issue here. I have something like,

    $settings: (general, profile, notification);

    @each $setting in $settings {
        &.list-item-#{$setting}-settings { background-image: url(../../img/client/icons/#{$setting}-settings.svg); }
    }

In the output, I get "false" instead of "notification".

**Comment by xzyfer** [comment]
Spec added https://github.com/sass/sass-spec/pull/255

**Comment by xzyfer** [comment]
This is fixed and will be in 3.2.
